src/services/subject-service.ts

```diff
@@ -4,14 +4,30 @@ import { AxiosResponse } from 'axios'
 import { URLs } from '~/constants/request'
 import { ItemsWithCount, SubjectInterface, SubjectNameInterface } from '~/types'
 import { createUrlPath } from '~/utils/helper-functions'
+import { baseService } from '~/services/base-service'
+import { getFullUrl } from '~/utils/get-full-url'
 
 export const subjectService = {
-  getSubjects: (
-    params?: Pick<SubjectInterface, 'name'>,
-    categoryId?: string
-  ): Promise<AxiosResponse<ItemsWithCount<SubjectInterface>>> => {
-    const category = createUrlPath(URLs.categories.get, categoryId)
-    return axiosClient.get(`${category}${URLs.subjects.get}`, { params })
+  getSubjects: (params?: Pick<SubjectInterface, 'name' | 'category'>) => {
```

Comment:
Why `params` is optional?
```suggestion
  getSubjects: (params: Pick<SubjectInterface, 'name' | 'category'>) => {
```

Reply:
Yes, it shouldn't be optional, thank you. Fixed.